package.json

```diff
@@ -12,11 +12,10 @@
     "make-manifest": "./bin/make-manifest"
   },
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 0",
     "lint": "eslint .",
     "snyk-protect": "snyk protect",
     "prepublish": "npm run snyk-protect",
-    "precommit": "npm run lint"
+    "prepare": "husky install"
```

Comment:
With `husky install` we don't need `npm run lint` anymore?

Reply:
We do. I've added it in the pre-commit hook (.husky/pre-commit). Is that what you mean @Betisman ?

Reply:
oh, ok, I missed it. 

Thanks